### ISSUE TITLE
Package Explorer crashes on startup caused by FileNotFoundException

### ISSUE THREAD

**Issue by jh-isw** [admin-shell-io/aasx-package-explorer#309]
**Describe the bug**
Application starts, shows Welcome Screen and then terminates without any message.

**To Reproduce**
1. Download AASX Package Explorer 2020-12-03.alpha (bigger file)
2. Right-click on zip-file -> Properties -> Unblock content from web
3. Extract folder
4. Start AasxPackageExplorer.exe

**Expected behavior**
Clear startup of the Application.

**Desktop (please complete the following information):**
 - OS: Windows 10 Version 1809 (OS Build 17763.1577)

**Additional context**
I looked into the Windows Event Viewer and found the error:

Log Name:      Application
Source:        .NET Runtime
Date:          12/6/2020 12:12:38 PM
Event ID:      1026
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      test-machine
Description:
Application: AasxPackageExplorer.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.IO.FileNotFoundException
   at CefSharp.Wpf.ChromiumWebBrowser.NoInliningConstructor()
   at CefSharp.Wpf.ChromiumWebBrowser..ctor()
   at AasxIntegrationBase.AasxPlugin.ActivateAction(System.String, System.Object[])

Exception Info: System.Reflection.TargetInvocationException
   at System.RuntimeMethodHandle.InvokeMethod(System.Object, System.Object[], System.Signature, Boolean)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(System.Object, System.Object[], System.Object[])
   at System.Reflection.RuntimeMethodInfo.Invoke(System.Object, System.Reflection.BindingFlags, System.Reflection.Binder, System.Object[], System.Globalization.CultureInfo)
   at AasxPackageExplorer.Plugins+PluginInstance.BasicInvokeMethod(System.String, System.Object[])
   at AasxPackageExplorer.BrowserContainer.Start(System.String, Boolean, Boolean)
   at AasxPackageExplorer.MainWindow.Window_Loaded(System.Object, System.Windows.RoutedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(System.Object, System.Windows.RoutedEventArgs, Boolean)
   at System.Windows.UIElement.RaiseEventImpl(System.Windows.DependencyObject, System.Windows.RoutedEventArgs)
   at System.Windows.BroadcastEventHelper.BroadcastEvent(System.Windows.DependencyObject, System.Windows.RoutedEvent)
   at System.Windows.BroadcastEventHelper.BroadcastLoadedEvent(System.Object)
   at MS.Internal.LoadedOrUnloadedOperation.DoWork()
   at System.Windows.Media.MediaContext.FireLoadedPendingCallbacks()
   at System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()
   at System.Windows.Media.MediaContext.RenderMessageHandlerCore(System.Object)
   at System.Windows.Media.MediaContext.RenderMessageHandler(System.Object)
   at System.Windows.Interop.HwndTarget.OnResize()
   at System.Windows.Interop.HwndTarget.HandleMessage(MS.Internal.Interop.WindowMessage, IntPtr, IntPtr)
   at System.Windows.Interop.HwndSource.HwndTargetFilterMessage(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   at MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)

I thought it could have to do with options-debug.MIHO.json so I removed it. Error persists.

Any Ideas how to investigate further?

**Comment by mristin** [comment]
Hi @jh-isw ,
I'll try to at least reproduce the error tomorrow (Monday) and come back to you.

**Comment by mristin** [comment]
Hi @jh-isw ,
The program runs without problems on my machine. Could you please try to start it from a console and post us the output?

**Comment by jh-isw** [comment]
Hi @mristin ,
I tried to start it from cmd and PowerShell. It goes into background immediately, no output.
The GUI starts and from there -> same process as described above. Terminates without a clue.

I have to add: I'm trying this on a fresh Windows 10. No developer tools or anything.
Unfortunately, it's not telling which file it was looking for.

Do you have the option to reproduce this on a clean Windows 10?

**Comment by mristin** [comment]
Hi @jh-isw ,
I only have my laptop at disposal. Could you try to run it through command prompt?

**Comment by mristin** [comment]
Could you please try to run it on another computer as well?

**Comment by jh-isw** [comment]
Hi @mristin ,
that's what I was doing originally and it didn't work. I installed Visual Studio and wanted to pull the source, but thought it might run now - and it did.

Slightly different Windows 10 Version. 2004 or 20H2.

This here was a cross-check and report of the finding. I think something is missing in the deployment, that usually comes with other software and is missing on a clean Windows 10.

**Comment by mristin** [comment]
@jh-isw Huh, I have no clue what could cause such behavior. @MichaelHoffmeisterFesto any thoughts?

**Comment by jh-isw** [comment]
I tested again with a clean Win 10 ( Version 2004 version since I don't have the 1809 anymore) and aasx-package-explorer.2021-02-25.alpha.

Initial observation persists (application not starting up), but different Error. 
Exception Info:
System.IO.FileNotFoundException at CefSharp.Wpf.ChromiumWebBrowser.NoInliningConstructor()
at CefSharp.Wpf.ChromiumWebBrowser..ctor()
at AasxIntegrationBase.AasxPlugin.ActivateAction(System.String, System.Object[])

Solution:
I installed the [Visual C++ 2019 Redistributables](https://support.microsoft.com/de-de/topic/aktuelle-unterst%C3%BCtzte-downloads-f%C3%BCr-visual-c-2647da03-1eea-4433-9aff-95f26a218cc0) as x64-version.

Result:
AasxPackageExplorer working as expected.
